packages/app/package.json

```diff
@@ -163,6 +163,7 @@
     "codesandbox-import-utils": "2.1.2",
     "color": "^0.11.4",
     "compare-versions": "^3.1.0",
+    "console": "^0.7.2",
```

Comment:
Is this dependency required? I can't see it being used anywhere

Reply:
jest-each@24.8.0 has a dependency on other jest modules and looks like it wasn't resolved automatically as result exception:

```
Failed to compile.

.../codesandbox-client/node_modules/@jest/console/build/BufferedConsole.js 
Module not found: Error: 
Can't resolve 'console' in ‘.../codesandbox-client/node_modules/@jest/console/build'
```
code wich cause this exeption:
```
function _console() {
  const data = require('console');

  _console = function _console() {
    return data;
  };

  return data;
}
```

I assume the safest way to solve this issue will be adding this dependency.

Reply:
I agree! Let's merge this in 😄.